src/utils/PayloadValidation.ts

```diff
@@ -1,11 +1,14 @@
 import {hgvValidation} from "./HgvValidations";
 import ITechRecord from "../../@Types/ITechRecord";
-import {VEHICLE_TYPE, SEARCHCRITERIA} from "../assets/Enums";
+import {VEHICLE_TYPE, SEARCHCRITERIA, ERRORS} from "../assets/Enums";
 import Joi, {ObjectSchema} from "@hapi/joi";
 import {psvValidation} from "./PsvValidations";
 import {trlValidation} from "./TrlValidations";
 import {validateOnlyAdr} from "./AdrValidation";
 import Configuration from "./Configuration";
+import {lgvValidation} from "./LgvValidations";
```

Comment:
Maybe we can have a folder here containing an index.ts 
that will export all the validations 

export * from './LgvValidations'
...
so we save a few more lines and make the import concise

then, in here instead of importing them individually we can do something like that:
import * as fromValidation from './validation-types'

then in the code we do:
fromValidation.lgvValidation.validate()